conda-store-server/conda_store_server/_internal/server/views/registry.py

```diff
@@ -79,6 +81,10 @@ def replace_words(s, words):
 
 def get_docker_image_manifest(conda_store, image, tag, timeout=10 * 60):
     namespace, *image_name = image.split("/")
+    response_headers = {
+        "Deprecation": "True",
+        "Sunset": "Mon, 16 Feb 2025 23:59:59 UTC",
+    }
```

Comment:
An alternative to this might be to make a `DeprecatedResponse` object which includes the correct response headers, and either wrap the current responses with it or make a decorator that does this automatically. That would help avoid typos in these keys, but I don't know if implementing deprecation machinery like that is worth the effort.

Reply:
Oh ya, I like this approach. I added just a decorator that adds the two deprecation headers. Could also extend this to create a `DeprecatedResponse` if you think that's more appropriate.